### DISCUSSION TITLE
choosing the output location

### DISCUSSION THREAD

**Discussion by raisfeld-ori** [PyO3/maturin#1702] [Q&A]
i am working over a python module with both rust and python. maturin has worked great, but i want to (eventualy) upload my package, in order to do that, i need to add the actual output of the code into the python project. i am able to do this by taking the maturin output out of the venv and placing it in the actual directory for the project. but it's annoying to develop and then place it in the main directory, since i often forget to actually place it. i know that this is probably possible to automate using the cargo.toml file, but i don't know how to do that, can anyone tell me if there's a method to do that?

**Comment by messense** [comment]
Have you tried `maturin build` or `maturin publish`?
